Exercises/5-chaining.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+// Check 4 digit pin.
+const checkPin = (ch1, ch2, ch3, ch4) =>
```

Comment:
```suggestion
const EXPECTED_PIN = '4967';
const expect = code => (...args) => args.join('') === code;
const checkPin = expect(EXPECTED_PIN);
```

Reply:
I thinks the suggested change is to atomic, makes the check visually longer and therefore increases an effort required to understand the task.

How about this version?

```js
const CORRECT_PIN = '4967';
const checkPin = (...code) => code.join('') === CORRECT_PIN;
```

Reply:
Fine